homeassistant/components/switcher_kis/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration uses entity services.
+  appropriate-polling:
+    status: exempt
+    comment: The integration does not poll.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: todo
+    comment: make sure flows end with created entry or abort
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup:
+    status: exempt
+    comment: devices are setup asynchronously and marked as unavailable until they are ready.
+  unique-config-entry:
+    status: exempt
+    comment: The integration only supports a single config entry.
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: todo
+  reauthentication-flow: done
+  test-coverage: done
+
+  # Gold
+  devices: done
+  diagnostics: done
+  discovery-update-info: done
```

Comment:
We don't really update the info based on the network right?

Reply:
We do, devices IP address are updated automatically from broadcasts by the devices

Reply:
But we don't update the config entry right?

Reply:
IPs are not stored in the config entry as they are automatically detected each time.
Explanation (this will answer https://github.com/home-assistant/core/pull/141065#discussion_r2056183817 also):
Device sends a broadcast every 4 second with all the data of the device so there is nothing to store, when the integration loaded it is dynamically creating devices as they are discovered. The IP of the device is also included in this packet (inside the payload, not the IP address of the packet) and this is what used to control the device.